src/main/webapp/index.html

```diff
@@ -19,7 +19,7 @@
         <div class="clear-icon">x</div>
       </div>
       <div class="loginButton">
-        <button id="logIn" type="button">Login</button>
+        <button id="login" data-text-swap="Logout">Login</button>
```

Comment:
**Important Style**
Login buttons and values will be referred to without camelCasing. @kcirerick 
``` login ```

Reply:
Good catch. Also, clever use of data-text-swap Claudia!